public/locales/ar/translation.json

```diff
@@ -638,14 +649,6 @@
         }
     },
     "wine": {
-        "actions": "الإجراء",
-        "manager": {
-            "link": "مدير واين",
-            "not-found": "لم يُعثر على إصدارات واين. انقر على رمز التحديث للمحاولة مرةً أخرى.",
-            "title": "مدير واين",
-            "unzipping": "جارٍ فك الضغط"
-        },
-        "release": "تاريخ الإصدار",
-        "size": "الحجم"
```

Comment:
I'm not sure how to handle this, but we are losing all the translations because of the main key changing from `wine` to `tool`

I wonder if there's a way to preserve them, I'm not really sure

Reply:
maybe they just have to change and that's it